tests/unit/components/NcAppNavigation/NcAppNavigation.spec.js

```diff
@@ -133,9 +133,9 @@ describe('NcAppNavigation.vue', () => {
 		})
 	})
 
-	describe('focus trap', () => {
-		// TODO
-	})
+	// describe('focus trap', () => {
+	// 	// TODO
+	// })
```

Comment:
I forgot to do it ><

By the way, it is a problem for Vitest to not comment it?
To avoid conflict on backport from `master` later.

Reply:
> By the way, it is a problem for Vitest to not comment it?

Unfortunately, yes. This is why I saw it 🙈 The test will fail with `test not found`.